axis.go

```diff
@@ -355,12 +350,22 @@ type DefaultTicks struct{}
 
 var _ Ticker = DefaultTicks{}
 
+var labels []float64
+
+var new_labels []float64
+
+var displayPrecision int
+
 // Ticks returns Ticks in a specified range
 func (DefaultTicks) Ticks(min, max float64) (ticks []Tick) {
 	const SuggestedTicks = 3
-	if max < min {
+	labels = nil
+	ticks = nil
```

Comment:
`ticks` is already `nil` at this point.
just drop that line.

Reply:
Changed that. Thanks!